router/step.go

```diff
@@ -39,6 +40,8 @@ func StepHandlers(base *gin.RouterGroup) {
 			step.PUT("", perm.MustPlatformAdmin(), middleware.Payload(), api.UpdateStep)
 			step.DELETE("", perm.MustPlatformAdmin(), api.DeleteStep)
 
+			step.POST("/stream", api.PostStepStream)
```

Comment:
I wonder if we should explore locking down this endpoint behind some permission layer?

Thinking for the other endpoints we call `perm.MustPlatformAdmin()` so perhaps we adopt that?

```suggestion
			step.POST("/stream", perm.MustPlatformAdmin(), api.PostStepStream)
```

Reply:
👍🏼